### DISCUSSION TITLE
No response from GitHub support

### DISCUSSION THREAD

**Discussion by AliMilani13** [community/community#142541] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

my original account was flagged as spam [Ali-Milani-13](https://github.com/Ali-Milani-13) I filed a ticket from that account but it has been 2 months and no one answered. 
same for this account. it has been month now. 

no warrant, no explanation, just suddenly flagged me out of nowhere and there's no response. 

what should I do?

**Comment by Mohsiareej** [comment]
Yu

**Comment by Aryma-f4** [comment]
It sounds very frustrating to be flagged without clear communication or response. The best steps you can take are:

- Follow Up on the Ticket: If your original ticket hasn’t been answered, you could try following up. Sometimes support queues get busy, and follow-ups can bring attention to your case.

- Submit a New Support Request: If it’s been over a month with no response, submitting a new support ticket for both accounts may help, especially if you reference your original ticket numbers.

- Check for Alternative Support Channels: If there’s a community forum, email support, or social media support channel, sometimes reaching out there with your ticket number can help expedite the process.

- Document Your Actions: Keep track of the dates you submitted tickets and any follow-up actions so you can present a clear case if you need further escalation.

**Reply by Mohsiareej** [reply]
Now I really don't know what can I do Next But That's Clear I Have Real One Ticket

**Reply by AliMilani13** [reply]
unfortunately there's no action. 

I've tried everything you've mentioned before. follow up ticket, new ticket. and they reject account related issues when you send email to them. 


> We now require that new support requests be created using our Support website:
> 
> https://support.github.com/
> 
> Please use this website to search through resources that may help you find the solutions you are looking for or connect with our GitHub experts.
> 
> Thank you,
> GitHub Support
> 
> **Please do not reply to this email, as it will not be seen by our team.**


social media support channels closed their DM and don't response to replies. I also documented everything. sent them screenshots and samples. nothing happened.

**Comment by afindo** [comment]
same as me

**Comment by thomassiard** [comment]
If your GitHub account has been flagged as spam and you haven't received a response for months, here are some steps you can take:

Check Spam or Junk Folders: Sometimes, GitHub's response may get filtered into spam/junk folders, so ensure you've checked all email folders.

Submit Another Ticket from a New Account: If you haven't already, try reaching out again using a different GitHub account or email to file a support ticket. Mention the case number of your previous request (if you have one) to expedite the process.

Use GitHub’s Social Media for Support Escalation: You can also try contacting GitHub Support through social media channels like [Twitter (X) @GitHubSupport](https://twitter.com/GitHubSupport). They may be able to assist in escalating your ticket.

Provide All Necessary Details in the Ticket: When you file the ticket, ensure you include all relevant details to prove your account's legitimacy, such as:

The username(s) of the flagged accounts.
Description of the activities that led up to the flagging.
Any additional information about your account’s use case (e.g., business-related, specific projects).

**Reply by AliMilani13** [reply]
unfortunately it doesn't work. I've tried every way you've suggested before. nothing happened. at this point maybe people must start hosting their own version control rather than trusting such broken bot moderate system. 

if they've valued their user even a bit, they would at least consider sending email according to their decision to describe what happened. but nothing. 

we don't reply to tickets. forget about email and no customer support number/chat.
